androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/PlayerControls.kt

```diff
@@ -40,28 +49,49 @@ fun PlayerControls(
         exit = fadeOut(),
         modifier = Modifier.testTag("PlayerControlsParent")
     ) {
-        Row(
-            horizontalArrangement = Arrangement.SpaceEvenly,
-            verticalAlignment = Alignment.CenterVertically,
+        Column(
             modifier = customization.modifier
-                .background(Color.Black.copy(alpha = 0.6f))
                 .fillMaxSize()
+                .background(Color.Black.copy(alpha = 0.6f))
         ) {
-            IconButton(onClick = onPreviousClick) {
-                customization.previousIconContent()
-            }
-            IconButton(
-                onClick = onPauseToggle,
-                modifier = Modifier.testTag("PauseToggleButton")
+            Row(
+                horizontalArrangement = Arrangement.SpaceEvenly,
+                verticalAlignment = Alignment.CenterVertically,
+                modifier = Modifier
+                    .fillMaxWidth()
```

Comment:
I will have problems with this `Modifier.fillMaxWidth` , because I need the rest of the screen for the forward/backward functionalities, I think this "Row" should only occupy a minimum required fixed width and the height should be the height of the child elements